homeassistant/components/mullvad/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+        },
+        "error": {
+            "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+            "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+            "unknown": "[%key:common::config_flow::error::unknown%]"
+        },
+        "step": {
+            "user": {
```

Comment:
Because this integration asks no data, it is kinda weird in the config flow, as it is empty:

![image](https://user-images.githubusercontent.com/195327/108846882-5e965780-75df-11eb-84dc-9e66eca4a80b.png)


```suggestion
            "user": {
               "description": "Set up the Mulvad VPN integration?",
```

We can use the description to work around that:

![image](https://user-images.githubusercontent.com/195327/108847118-969d9a80-75df-11eb-9bdd-063d2ef68f8a.png)